### PR TITLE
New UI

### PR DESCRIPTION
**Pull request by arbales**

### CODE REVIEW

**Comment by arbales** [comment]
**TODO**
- Move controls atop album art
- Make queue scroll independently
- Re-add usernames or facepiles or twipsies

**Comment by arbales** [comment]
ping @nixme

File: push/package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
   "name": "warble-push",
   "description": "Push components of the Warble jukebox server",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "author": "Gopal Patel <nixme@stillhope.com>",
   "engines": {
-    "node": ">= 0.6.2"
+    "node": ">= 0.6.2",
+    "npm": "~1.1.18"
```
Comment (arbales) [inline push/package.json]: There have been some NPM changes recently, you might need to update the node version being used on Alderaan

**Comment by arbales** [comment]
:(

**Comment by nixme** [comment]
Alright, I'll merge this and your other branch together. Turn that frown upside down.

File: app/assets/javascripts/application.js.coffee [hunk 2]
```diff
@@ -56,30 +61,24 @@ jQuery(document).ready ($) ->
...
       $(@currentView.el).remove() if @currentView
+      @serviceChooserView.autoSelectTab()
       @paneEl.append view.render().el
       view.delegateEvents()   # TODO: make this unnecessary
       view.activate?()
       @currentView = view
 
     home: ->
-      @switchPane @serviceChooserView
+      @switchPane new Backbone.View
```
Comment (arbales) [inline app/assets/javascripts/application.js.coffee:80]: Placeholder

File: app/assets/stylesheets/application.css.sass [hunk 3]
```diff
@@ -2,10 +2,19 @@
...
-
+@import 'effects'
+@import 'spinner'
+@import 'slider'
 @import '_tooltips'
 
+$left-column-width: 320px
+$small-artwork-size: 32px
+$card-style-padding: 20px
+$default-font: Roboto, 'Helvetica Neue', HelveticaNeue, Arial, Helvetica, sans-serif
```
Comment (nixme) [inline app/assets/stylesheets/application.css.sass:13]: Who's gonna have Roboto installed? Web font it?

File: app/controllers/votes_controller.rb [hunk 4]
```diff
@@ -3,14 +3,17 @@ class VotesController < ApplicationController
 
   def create
     begin
-      if (song = Song.find(params[:song_id]).votes.create(user: current_user)).save
+      song = Song.find(params[:song_id])
+      vote = song.votes.build()
```
Comment (nixme) [inline app/controllers/votes_controller.rb:7]: no parens

**Comment by nixme** [comment]
Couple minor things, then merge it!

**Pull request merged**